datakyt/db/dump_db.py

```diff
@@ -0,0 +1,36 @@
+import subprocess
+import logging
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG,
+                    format='%(asctime)s.%(msecs)03d[%(process)d:%(thread)d:%(name)s:%(lineno)d] %(levelname)s %('
+                           'message)s',
+                    datefmt='%Y-%m-%d %I:%M:%S')
+
+
+def backup_postgres_db(host, database_name, port, user, password, dest_file):
+    """
+  Backup postgres db to a file.
+  """
+    try:
+        process = subprocess.Popen(
+            ['pg_dump',
+             '--dbname=postgresql://{}:{}@{}:{}/{}'.format(user, password, host, port, database_name),
+             '-Fc',
+             '-f', dest_file,
+             '-v'],
+            stdout=subprocess.PIPE
+        )
+        output = process.communicate()[0]
```

Comment:
Check what's `output` here and why do we return it.

Reply:
I think we don't need this output here. If we fail to connect to the database, the logger will show us this in if.
So maybe I will delete 2 lines,  output and return.